hw/ip/sram_ctrl/data/sram_ctrl_sec_cm_testplan.hjson

```diff
@@ -40,6 +40,17 @@
       stage: V2S
       tests: ["{name}_regwen"]
     }
+    {
+      name: sec_cm_readback_config_regwen
+      desc: '''Verify the countermeasure(s) READBACK.CONFIG.REGWEN.
+
+            The `readback` CSR is excluded in CSR tests, add another test to verify:
+             - When `readback_regwen` is 1, writting to `readback` can take effect.
```

Comment:
Nit.
```suggestion
             - When `readback_regwen` is 1, writing to `readback` can take effect.
```

Reply:
Ah thanks, now also fixed in the `sec_cm_ctrl_config_regwen` description above.